3 - FightClub/Code/Objects/Boxer.cs

```diff
@@ -0,0 +1,94 @@
+﻿using ObjectOrientedProblems.Code.Enums;
+using ObjectOrientedProblems.Code.Interfaces;
+
+namespace ObjectOrientedProblems.Code
+{
+    internal class Boxer : IFighter
+    {
+        public Boxer()
+        {
+            Damage = 1;
+            Health = 10;
+            State = FighterState.Healthy;
+        }
+        public int Health { get; set; }
+
+        public int Damage { get; set; }
+
+        public FighterState State { get; set; }
```

Comment:
In C# properties are usually found *above* the constructor / other methods.  Can also be short handed into:

```
		public int Health { get; set; } = 10;

		public int Damage { get; set; } = 1;

		public FighterState State { get; set; } = FighterState.Healthy;
```